app/__init__.py

```diff
@@ -86,11 +91,22 @@ def setup_funds_and_auth(app: Flask) -> None:
     towns_fund: FundConfig = TOWNS_FUND_APP_CONFIG
     app.config["FUND_CONFIGS"] = FundService(role_to_fund_configs={towns_fund.user_role: towns_fund})
 
+    pathfinders: FundConfig = PATH_FINDERS_APP_CONFIG
+    app.config["FUND_CONFIGS"] = FundService(role_to_fund_configs={pathfinders.user_role: pathfinders})
+
     # auth
-    tf_auth = TOWNS_FUND_AUTH
+    tf_auth = FUND_AUTH
     tf_auth.update(Config.ADDITIONAL_EMAIL_LOOKUPS)
+
+    pf_auth = {}
+    for key, value in FUND_AUTH.items():
+        pf_auth[key] = (value[0], "Pathfinders")
```

Comment:
We don't need to add `Pathfinders` here - it should just be 
```
pf_auth[key] = (value[0],)
```

Reply:
It would be clearer to write it as
```
pf_auth = {}
for domain, (local_authorities, places, fund_types) in FUND_AUTH.items():
    pf_auth[domain] = (local_authorities,)
```

Reply:
I think there's probably a clearer way of defining these though, like:

```
DOMAIN_TO_LA_PLACES = {
"example.gov.uk": ((local_authorities,),(places,))
}  # this would include all domains specified in the current TF auth mapping

TF_AUTH = {domain: (las, places, {"Town_Deal", "Future_High_Street_Fund"} for domain, (las, places) in DOMAIN_TO_LA_MAPPING.items())

PF_AUTH = {domain: (las,} for domain, (las, _) in DOMAIN_TO_LA_MAPPING.items()}
```

Reply:
Could we move the setup of the auth dictionaries to config and implement it like this instead so that both are based off of a domain to LA and place mapping. Then in the application factory we can just import `TF_AUTH` and `PF_AUTH`